fusor-ember-cli/app/components/node-profile.js

```diff
@@ -25,7 +23,8 @@ export default Ember.Component.extend({
     var profile = this.get('profile');
     var params = this.get('plan.parameters');
     var self = this;
-    this.get('plan.roles').forEach(function(role) {
+    var roles = this.get('plan.roles') || [];
+    roles.forEach(function(role) {
```

Comment:
I was getting errors that `this.get('plan.roles')`  returned undefined, so I added this although it should have returned `[ ]` if empty. Not sure why????